src/tools/pretty.ts

```diff
@@ -1,12 +1,36 @@
-import { print } from "gluegun/print"
+import { print } from "gluegun"
```

Comment:
Was this not working with `gluegun/print`? If so, that might be a bug with Gluegun.

Reply:
It works but is untyped. I can switch it back but I thought it was nice to have the type safety.

Reply:
Ooh, didn't realize it was untyped!! That's something we'll want to fix eventually on Gluegun. Thanks for pointing it out!